overlays/DIMO/dimo.vspec

```diff
@@ -66,3 +66,8 @@ Vehicle.Powertrain.TractionBattery.Charging.AddedEnergy:
   datatype: float
   unit: kWh
   description: Amount of charge added to the high voltage battery during the current charging session, expressed in kilowatt-hours.
+
+Vehicle.IgnitionStatus:
```

Comment:
[The style guide suggests](https://covesa.github.io/vehicle_signal_specification/rule_set/basics/#style-guide) that booleans start with `Is`. I think it probably leads to more easily understandable names. Like `IsIgnitionOn: true` reads better than `IgnitionStatus: true` to me.

Reply:
great, thanks for review, will update.